R/pstr_prepare_scenario.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @param scenarios A named list of identically structured scenarios.
 #'
-#' @family pre-processing helpers
+#' @family helpers
```

Comment:
@maurolepore  I would really like to know about how @family of function is useful during package development. Is it same as the inheritance concept between parent and child classes in Object Oriented Programming (OOO)?

Reply:
@kalashsinghal, great question:

* [Here is the explanation of what it does with R documentation](https://roxygen2.r-lib.org/articles/index-crossref.html#family).

* [Here is an example in our package (the section see-also is generated automatically with cross-references).](https://2degreesinvesting.github.io/tiltIndicator/reference/pstr_prepare_companies.html#see-also).

* [Here is what it does on the Reference section of the package website.](https://pkgdown.r-lib.org/reference/build_reference.html?q=family#topic-matching). Note:

> ... If you are using roxygen2, `has_concept()` also matches family tags, because roxygen2 converts them to concept tags.

* [Here is how we use it in the configuration file of the tiltIndicator website](https://github.com/2DegreesInvesting/tiltIndicator/blob/main/_pkgdown.yml).

* [Here is the resulting Reference in the tiltIndicator website](https://2degreesinvesting.github.io/tiltIndicator/reference/index.html).